tests/features/step_definitions/incident_steps.coffee

```diff
@@ -35,3 +43,38 @@ do ->
       groups = @client.elements(selector)
       if groups.value.length != 1
         throw new Error('ScatterPlot Group is empty')
+
+    @When /^I add the first suggested event source$/, ->
+      @client.clickWhenVisible('.open-source-form-in-details')
+      @client.waitForVisible('#event-source')
+      @client.clickWhenVisible('#suggested-articles li:first-child')
+      @client.setValue('input[name="publishTime"]', '12:00 PM')
+      @client.click('button.save-modal[type="button"]')
+
+    @When /^I add the first suggested incident report$/, ->
+      # SuggestedIncidentsModal
+      @client.waitForVisible('#suggested-locations-form p.annotated-content')
+      # make sure the element is within view (webdriver.io scroll doesn't work
+      # inside eidr-connect modal)
+      scrollWithinModal(@client,
+          '#suggestedIncidentsModal div.suggested-incidents-wrapper',
+          'span.annotation.annotation-text', -200)
+      # we do not know if the suggestedIncidentsModal will have any annotations
```

Comment:
So this test always runs we should use an article we are certain will have annotations.